src/Student.java

```diff
@@ -0,0 +1,30 @@
+public class Student{
+
+    private String name;
+    private String secondName;
+    private Long ID;
+    private String TypeOfStudy;
```

Comment:
As we have some specific types of studies -> a String is not the best option here, cause it indicates we could write anything we want in here, e.g. a "pizza"